packages/synapse-interface/store/store.ts

```diff
@@ -77,43 +77,45 @@ store.subscribe(() => {
     segmentAnalyticsEvent(eventTitle, eventData)
   }
 
-  if (
-    previousState.bridgeDisplay.showDestinationAddress === false &&
-    currentState.bridgeDisplay.showDestinationAddress === true
-  ) {
-    eventTitle = `[Bridge User Action] Show destination address`
-    eventData = {}
-
-    segmentAnalyticsEvent(eventTitle, eventData)
-  }
-
-  if (
-    previousState.bridgeDisplay.showDestinationAddress === true &&
-    currentState.bridgeDisplay.showDestinationAddress === false
-  ) {
-    eventTitle = `[Bridge User Action] Hide destination address`
-    eventData = {}
-
-    segmentAnalyticsEvent(eventTitle, eventData)
-  }
-
-  if (
-    previousState.bridgeDisplay.showSettingsSlideOver === false &&
-    currentState.bridgeDisplay.showSettingsSlideOver === true
-  ) {
-    eventTitle = `[Bridge User Action] Show Settings`
-    eventData = {}
-    segmentAnalyticsEvent(eventTitle, eventData)
-  }
-
-  if (
-    previousState.bridgeDisplay.showSettingsSlideOver === true &&
-    currentState.bridgeDisplay.showSettingsSlideOver === false
-  ) {
-    eventTitle = `[Bridge User Action] Hide Settings`
-    eventData = {}
-    segmentAnalyticsEvent(eventTitle, eventData)
-  }
+  /** TODO: Replace below segment events with new tracked events */
```

Comment:
Add these back

Reply:
thanks, added back in https://github.com/synapsecns/sanguine/commit/1226707ad784b0f397f14d47e45288cd56c7c4f1

previously commented these out because we initially were planning on removing Settings toggle